### PR TITLE
Change healthcheck API to report open circuits but return "GREEN" anyway #144

### PR DESCRIPTION
**Pull request by seratch**
refs #144  
I can't stand receiving noisy (and almost useless) mobile alerts anymore.. This pull request goes with the second suggestion by @mauhiz [here](https://github.com/m3dev/octoparts/issues/144#issue-63890829).

### CODE REVIEW

File: app/controllers/system/HealthcheckController.scala [hunk 1]
```diff
@@ -45,7 +45,9 @@ class HealthcheckController(configsRepo: ConfigsRepository,
           "hystrix" -> hystrixStatus,
           "memcached" -> memcachedStatus
         )
-        val colour = calculateColour(statuses.values)
+        val colour = calculateColour(
+          statuses.filter { case (key, _) => key != "hystrix" }.values
```
Comment (lloydmeta) [inline app/controllers/system/HealthcheckController.scala]: `filterKeys` might be nice here.

Comment (seratch) [inline app/controllers/system/HealthcheckController.scala]: Exactly! Thanks, to be honest, I didn't know that. I just fixed that.

**Comment by lloydmeta** [comment]
Oops, looks like [a test needs to be updated](https://travis-ci.org/m3dev/octoparts/jobs/57594580#L630)

**Comment by seratch** [comment]
I just fixed the test suite. I hope Travis builds will succeed this time.

**Comment by lloydmeta** [comment]
Sweet, thanks for this.

**Pull request merged**